parser.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import json
+import xml.etree.ElementTree as ET
+from collections import defaultdict
+
+
+def etree_to_dict(t):
```

Comment:
Do we even need this file anymore - this code has been refactored right, like in utils.py and parse_ccv.py?